cmd/flow-dps-indexer/main.go

```diff
@@ -54,13 +54,15 @@ func run() int {
 		flagIndex      string
 		flagLevel      string
 		flagTrie       string
+		flagForce      bool
```

Comment:
They are ordered alphabetically, so this one moves up to behind `flagData`. That's done so it's consistent with the help output we get on the shell.